plugin/storage/cassandra/schema.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+CQLSH_HOST=${CQLSH_HOST:-"cassandra"}
+CASSANDRA_WAIT_TIMEOUT=${CASSANDRA_WAIT_TIMEOUT:-"60"}
+DATACENTER=${DATACENTER:-"openshift"}
```

Comment:
The datacenter is actually required by the existing schema generator: https://github.com/uber/jaeger/blob/master/plugin/storage/cassandra/cassandra3v001-schema.sh#L23

So, it's either `prod` or `test`, and when passing `prod`, a datacenter has to be provided.